### PR TITLE
annalyns-infiltration concept exercise

### PR DESCRIPTION
**Pull request by colinleach**
`hints.md` is blank at present. Worth adding something?

### CODE REVIEW

**Review by jonmcalder — APPROVED** [review]
Thanks this looks good!

(one minor change I'd suggest but it's not crucial)

**Comment by jonmcalder** [comment]
No pressure to add any hints for now since they aren't essential and we can always add them later.

File: exercises/concept/annalyns-infiltration/.docs/instructions.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+You have four tasks: to implement the logic for determining if the above actions are available based on the state of the three characters found in the forest and whether Annalyn's pet dog is present or not.
+
+## 1. Check if a fast attack can be made
+
+Implement the `can_fast_attack()` method that takes a boolean value that indicates if the knight is awake. This method returns `TRUE` if a fast attack can be made based on the state of the knight. Otherwise, returns `FALSE`:
+
+```R
+knight_is_awake <- TRUE
+can_fast_attack(knight_is_awake)
+# <-> FALSE
```
Comment (jonmcalder) [inline exercises/concept/annalyns-infiltration/.docs/instructions.md:32]: This is different from the convention you've used for expressing function return values in later concept exercises.

I think that this pattern:
```
# => {return value}
```
that you've used in #256 and #260 is better.

File: exercises/concept/annalyns-infiltration/test_annalyns-infiltration.R [hunk 2]
```diff
@@ -0,0 +1,395 @@
+source("./annalyns-infiltration.R")
+library(testthat)
+
+context("annalyns-infiltration")
```
[the remark this answers is not in the document]
Comment (colinleach) [reply exercises/concept/annalyns-infiltration/test_annalyns-infiltration.R:4]: OK, I'd wondered about that but blindly followed the older exercises. I'll make this change in all the outstanding PRs, as well as new exercises going forward.

File: exercises/concept/annalyns-infiltration/.docs/instructions.md (hunk 1 above)
Comment (colinleach) [reply exercises/concept/annalyns-infiltration/.docs/instructions.md:32]: Aaargh! I failed to spot some auto-text that VS code had added. I'll fix it!

Full disclosure: several of these `instructions.md` files were copied from the Javascript track, and I just perpetuated their comment syntax (with # in place of //). Lazy...

**Comment by colinleach** [comment]
Changed as suggested.

**Pull request merged**